core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/BoxUse.java

```diff
@@ -36,4 +36,27 @@ public void setAlias(String alias) {
   public void setId(long id) {
     this.id = id;
   }
+
+  @Override
+  public int hashCode() {
+    final int prime = 31;
+    int result = 1;
+    result = prime * result + ((alias == null) ? 0 : alias.hashCode());
+    result = prime * result + (int) (id ^ (id >>> 32));
+    return result;
+  }
+
+  @Override
+  public boolean equals(Object obj) {
+    if (this == obj) return true;
+    if (obj == null) return false;
+    if (getClass() != obj.getClass()) return false;
+    BoxUse other = (BoxUse) obj;
+    if (alias == null) {
+      if (other.alias != null) return false;
+    } else if (!alias.equals(other.alias)) return false;
+    if (id != other.id) return false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1126)